packages/cli/src/commands/org/search/dump.ts

```diff
@@ -32,7 +38,9 @@ interface FetchParameters {
 }
 
 export default class Dump extends Command {
+  private static readonly numberOfResultPerQuery = 1000;
```

Comment:
Quick win might be to also make this a command flag, default to 1000

Would help with having to exclude a ton of fields when they hit API limits.

ie: instead of clients having to exclude a lot of field, ask them to fiddle with/lower that number.

Could also detect internally when we get a max size response, and automatically retry with something like `numberOfResultPerQuery = numberOfResultPerQuery / 2`

Reply:
I'm working on this, there's a Jira for that CDX-986.

I'm taking the 'automatic' approach.